mobx/lib/src/core/typedefs.dart

```diff
@@ -1,7 +1,7 @@
 part of '../core.dart';
 
 /// aliases for primitive types
-typedef ObservableInt = Observable<int>;
-typedef ObservableDouble = Observable<double>;
-typedef ObservableBool = Observable<bool>;
-typedef ObservableString = Observable<String>;
+typedef ObsInt = Observable<int>;
+typedef ObsDouble = Observable<double>;
+typedef ObsBool = Observable<bool>;
+typedef ObsString = Observable<String>;
```

Comment:
I don't see a need for these typedefs...Its better to expose the raw generic type directly...more readable that way

Reply:
I agree, i will remove these typedefs.
They just make a code more tangled.

Reply:
@pavanpodila done.

Reply:
@pavanpodila please fix Observer, it's not reacting to any observables

Reply:
@gabrielscr how does it relate to my MR?
Create your own issue and ask for help there.

Reply:
@subzero911 no